### DISCUSSION TITLE
Announce songs when they start playing.

### DISCUSSION THREAD

**Discussion by DesperatePea** [jagrosh/MusicBot#1394] [Questions]
I have not found a single info about this anywhere, but i feel like this is a pretty standard discord bot feature. I could be blind and havent found how to enable it, but it would be nice to have this as a feature. Its a great bot, works perfectly, but this is sorely needed.

**Comment by gacrestani** [comment]
Agreed. Also, print the queue (by running !queue) every time someone adds a song would be great.
